Source/bmalloc/libpas/src/libpas/bmalloc_heap_inlines.h

```diff
@@ -130,6 +130,37 @@ bmalloc_allocate_auxiliary_with_alignment_inline(pas_primitive_heap_ref* heap_re
     return bmalloc_allocate_auxiliary_with_alignment_casual(heap_ref, size, alignment, allocation_mode);
 }
 
+static PAS_ALWAYS_INLINE void*
+bmalloc_try_allocate_auxiliary_zeroed_with_alignment_inline(pas_primitive_heap_ref* heap_ref,
+                                                     size_t size,
+                                                     size_t alignment,
+                                                     pas_allocation_mode allocation_mode)
+{
+    pas_allocation_result result;
+    result = bmalloc_try_allocate_auxiliary_impl_inline_only(heap_ref, size, alignment, allocation_mode);
+    if (PAS_LIKELY(result.did_succeed))
+        return (void*)result.begin;
```

Comment:
Does this need to be zero'ed explicitly?  The equivalent case in `bmalloc_allocate_auxiliary_zeroed_with_alignment_inline` calls `pas_allocation_result_zero` on the result of `bmalloc_allocate_auxiliary_impl_inline_only`.  Is it not needed here?